core/src/main/java/org/apache/accumulo/core/spi/common/SystemContext.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.accumulo.core.spi.common;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.accumulo.core.client.TableNotFoundException;
+
+/**
+ * This interface exposes Accumulo system level information to plugins in a stable manner. The
+ * purpose of this interface is to insulate plugins from internal refactorings and changes to
+ * Accumulo.
+ *
+ * @since 2.0.0
+ */
+public interface SystemContext {
+
+  /**
+   * @since 2.0.0
+   */
+  public interface Configuration extends Iterable<Entry<String,String>> {
+
+    /**
+     * @return The value for a single property or null if not present. Sensitive properties are
+     *         intentionally not returned in order to prevent inadvertent logging of them. If your
+     *         plugin needs sensitive properties a getSensitive method could be added.
+     */
+    String get(String key);
+
+    /**
+     * Users can set arbitrary custom properties in Accumulo using the prefix
+     * {@code general.custom.}. This method will return all properties with that prefix, stripping
+     * the prefix. For example, assume the following properties were set :
+     *
+     * <pre>
+     *  {@code
+     *   general.custom.prop1=123
+     *   general.custom.prop2=abc
```

Comment:
I looked up the documention and the `@code` tag implies it will use the code font.  The only difference between the `@code` and `@literal` tag is that `@code` uses the code font.